src/cli/iofog.js

```diff
@@ -59,23 +63,22 @@ class IOFog extends BaseCLIHandler {
       { name: 'device-frequency', alias: 'Q', type: Number, description: 'ioFog node device scan frequency (seconds)', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'blutooth-enable', alias: 'B', type: Boolean, description: 'Enable bluetoth on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'blutooth-disable', alias: 'b', type: Boolean, description: 'Disable bluetoth on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
+      { name: 'bluetooth-enable', alias: 'B', type: Boolean, description: 'Enable bluetooth on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
+      { name: 'bluetooth-disable', alias: 'b', type: Boolean, description: 'Disable bluetooth on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
       { name: 'watchdog-enable', alias: 'W', type: Boolean, description: 'Enable watchdog on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
       { name: 'watchdog-disable', alias: 'w', type: Boolean, description: 'Disable watchdog on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'abs-hw-enable', alias: 'a', type: Boolean, description: 'Enable hardware abstraction on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'abs-hw-disable', alias: 'A', type: Boolean, description: 'Disable hardware abstraction on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'gps-enable', alias: 'p', type: Boolean, description: 'Enable GPS on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'gps-disable', alias: 'P', type: Boolean, description: 'Disable GPS on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
+      { name: 'abs-hw-enable', alias: 'A', type: Boolean, description: 'Enable hardware abstraction on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
+      { name: 'abs-hw-disable', alias: 'a', type: Boolean, description: 'Disable hardware abstraction on ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
       { name: 'reboot', alias: 'o', type: Boolean, description: 'Reboot ioFog node', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
       { name: 'fog-type', alias: 'y', type: Number, description: 'ioFog node architecture type', group: [constants.CMD_UPDATE, constants.CMD_ADD] },
-      { name: 'enable', alias: 'e', type: Boolean, description: 'Enable tunnel', group: [constants.CMD_TUNNEL] },
-      { name: 'disable', alias: 'S', type: Boolean, description: 'Disable tunnel', group: [constants.CMD_TUNNEL] },
-      { name: 'info', alias: 'O', type: Boolean, description: 'Display tunnel info', group: [constants.CMD_TUNNEL] },
+      { name: 'tunnel-enable', alias: 'e', type: Boolean, description: 'Enable tunnel', group: [constants.CMD_TUNNEL] },
```

Comment:
why did you add `tunnel-` to these? I think `enable`, `disable` and `info` are better.

The command will be:
`fog-controller iofog tunnel --enable`